packages/backend/src/nest/storage/certifacteRequests/certificatesRequestsStore.ts

```diff
@@ -102,7 +102,8 @@ export class CertificatesRequestsStore extends EventStoreBase<string> {
           const pubKey = keyFromCertificate(parsedCsr)
 
           if (filteredCsrsMap.has(pubKey)) {
-            filteredCsrsMap.delete(pubKey)
+            this.logger.warn(`Skipping csr due to existing pubkey`, pubKey)
```

Comment:
The event store now returns records in descending order rather than ascending order.